src/pybind/mgr/telemetry/module.py

```diff
@@ -1751,6 +1761,9 @@ def self_test(self) -> None:
         if 'report_id' not in report:
             raise RuntimeError('report_id not found in report')
 
+        # Ensure that triggering "send" does not break anything in the module
+        self.send(report)
```

Comment:
this should be removed since the user can just experiment with the `selftest` module, and their reports will be sent unknowingly, without agreeing to the license

Reply:
Good point. Would you recommend adding a test elsewhere? As you said, it's a bit tricky to do without calling the send() function to send a report.

Reply:
yes, the thing is this means all tests will pollute the db; let's hold on with that in the meantime until we have a sustainable way of testing this